src/controllers/projectController.js

```diff
@@ -1,118 +1,159 @@
 /* eslint-disable quotes */
 /* eslint-disable arrow-parens */
-const mongoose = require("mongoose");
-const timeentry = require("../models/timeentry");
-const userProfile = require("../models/userProfile");
-const userProject = require("../helpers/helperModels/userProjects");
-const { hasPermission } = require("../utilities/permissions");
-const escapeRegex = require("../utilities/escapeRegex");
-const cache = require("../utilities/nodeCache")();
+const mongoose = require('mongoose');
+const timeentry = require('../models/timeentry');
+const userProfile = require('../models/userProfile');
+const userProject = require('../helpers/helperModels/userProjects');
+const helper = require('../utilities/permissions');
+const escapeRegex = require('../utilities/escapeRegex');
+const cache = require('../utilities/nodeCache')();
 
 const projectController = function (Project) {
   const getAllProjects = function (req, res) {
-    Project.find({}, "projectName isActive category modifiedDatetime")
+    Project.find({}, 'projectName isActive category modifiedDatetime')
       .sort({ modifiedDatetime: -1 })
       .then((results) => {
         res.status(200).send(results);
       })
-      .catch((error) => res.status(404).send(error));
+      .catch((error) => res.status(404).send(error.message));
```

Comment:
I don't think this is safe. Why would you return the `message` string rather than the Error itself? This will break the frontend app right?